IntegrationTests/Braintree-API-Integration-Specs/BTAPIClient_IntegrationTests.m

```diff
@@ -16,7 +16,7 @@ - (void)testFetchConfiguration_withTokenizationKey_returnsTheConfiguration {
         [expectation fulfill];
     }];
 
-    [self waitForExpectationsWithTimeout:5 handler:nil];
+    [self waitForExpectationsWithTimeout:10 handler:nil];
```

Comment:
These tests are for a network call to `fetchOrReturnRemoteConfiguration` - since we are making a network call in this test, we can bump the timeout to allow for the networking later to return a result.

Reply:
Yeah, agree it shouldn't take that long - I do think we should seriously consider exploring self hosted runners. This is a bandaid fix but the issue ultimately seems to lie somewhere in the GitHub action runner resources.

Reply:
Yeh - we should look into other iOS specific CI offerings like BitRise and CircleCI. I think a custom solution would be a ton of work and most companies have entire teams dedicated to that infrastructure. I would love to learn more about what resources Venice & Venmo use 👀